WcaOnRails/config/locales/en.yml

```diff
@@ -1216,7 +1216,7 @@ en:
       #context: WEC
       wec:
         name: "WCA Ethics Committee"
-        description: "This committee has the role to ensure that all the procedures followed by the WCA Staff and External Staff abide by its Code of Ethics."
+        description: "This committee has the role to ensure that all the procedures followed by the WCA Staff and External Staff abide by its Code of Ethics, and to perform independent investigations if the conduct of any of these members is not compliant with the said Code of Ethics. Such investigations can be requested by any Registered Speedcubers, for whom the WEC is a confidant. This committee also functions as the board of appeal if anyone involved disagree with a decision made by the WCA Disciplinary Committee."
```

Comment:
disagree -> disagrees?

How do you feel about linking to the Code of Ethics here?

"This committee's role is..." sounds more natural to me than "This committee has the role to..."

Reply:
I would also change "Registered Speedcubers" to "WCA members." That's the "official" term of what you want to say and makes it clear that you have to be registered with the WCA, not any speedcubers forum. :smile:

Reply:
Well, for this one I would like to stick to the terms that are officially defined by the By-Laws.
We could add something into brackets tough: "(anyone who competed in a WCA Competition)".

Reply:
Oh, yes, sorry. Of course, that makes sense. I didn't remember that's a term we have in the by-laws now.